### PR TITLE
[CI] automate release process

### PR DESCRIPTION
**Pull request by v1v**
### What

Update docs with the contributing guideline. Add release guideline.
Enable release automation when a tag release is created with the format `d+.\d+.\d+`
Release automation:
* will create a release in github matching the tag.
* will create a changelog using the `gren` app, filtering only the Pull Requests with certain labels:
  * "Breaking changes" category for all the PRs with the `breaking` label.
  * "Bug fixes" category for all the PRs with the `bug` or `fix` labels.
  * "Features" category for all the PRs with the `enhancement`, `feature` or `feat` labels.

### Actions

- [x] Validate scripts locally

### How to run locally

```bash
$ GITHUB_TOKEN=${GREN_GITHUB_TOKEN} TAG_NAME=1.0.2 make -f .ci/Makefile release
```

Output

```bash
Getting gren config from local file /app/.ci/.grenrc.js

🤖  - Generate release notes:
===================================
✔ Releases found: 3
✔ Tags found: 1.0.2
✔ Pull Requests found: 13
✔ 1.0.2 has been successfully created!
See the results here: https://github.com/elastic/ecs-logging-php/releases/tag/1.0.2
```

### Tests

image

### CODE REVIEW

**Comment by v1v** [comment]
Somehow the current implementation with the `gren` library seems to be a bit buggy, so I'll try something else then

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Releasing
+
+## CI based
+
+The release process is also automated in the way any specific commit, from the master branch ideally, can be potentially released, for such it's required the below steps:
+
+1. Checkout the commit to be released
+1. Create a tag with the format `v.[0-9]+.[0-9]+.[0-9]+`, i.e.: `v1.0.2`
```
Comment (v1v) [inline RELEASE.md:8]:
```suggestion
1. Create a tag with the format `[0-9]+.[0-9]+.[0-9]+`, i.e.: `1.0.2`
```

**Review by SergeyKleyman — APPROVED** [review]
LGTM

File: .ci/docker/gren/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM node:14.5.0-stretch-slim
```
Comment (kuisathaverat) [inline .ci/docker/gren/Dockerfile:1]: Could we use this image in other places? if so, I'd move it to the library in a follow-up a publish the Docker container in our registry.

**Review by kuisathaverat — APPROVED** [review]

**Pull request merged**